modules/core/src/main/scala/gem/parser/Misc.scala

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.parser
+
+import atto._, Atto._
+import scala.annotation.tailrec
+
+/** General-purpose parsers and combinators that aren't provided by atto. */
+trait MiscParsers {
+
+  /** Parser for a hyphen. */
+  val hyphen: Parser[Unit] =
+    char('-').void
+
+  /** Parser for one or more spaces. */
+  val spaces1: Parser[Unit] =
+    skipMany1(char(' '))
+
+  /** Parser for a non-whitespace string. */
+  val nonWhitespace: Parser[String] =
+    stringOf(elem(c => !c.isWhitespace))
```

Comment:
Yeah, actually there's a more efficient combinator I can use here. `takeWhile` will look at input chunk by chunk instead of char by char.